xml_converter/src/xml_converter.cpp

```diff
@@ -168,10 +185,27 @@ void process_data(
     for (size_t i = 0; i < input_guildpoint_paths.size(); i++) {
         cout << "Loading guildpoint pack " << input_guildpoint_paths[i] << endl;
 
-        read_burrito_directory(
+        set<string> category_names = read_burrito_directory(
             input_guildpoint_paths[i],
             &marker_categories,
             &parsed_pois);
+        for (string category_name : category_names) {
+            if (find(top_level_categories.begin(), top_level_categories.end(), category_name) != top_level_categories.end())
+                duplicate_categories.push_back(category_name);
+            else
+                top_level_categories.insert(category_name);
+        }
+    }
+
+    if (duplicate_categories.size() > 0) {
+        cout << "The following top level categories were found in more than one pack" << endl;
+        for (size_t i = 0; i < duplicate_categories.size(); i++) {
+            cout << duplicate_categories[i] << endl;
+        }
```

Comment:
Do we want to print out all the duplicates if we are allowing for duplicates? Also just checking top-level categories means that we will only detect top level category duplicates. Which is enough to answer "are there duplicates" but not "what duplicates are there".

Reply:
That's a good point. I will move the check. 

I think we only want to check top level duplicates. If there are any duplicates in the children than the top level will be duplicated as well. If for instance we are running a pack twice, the list of duplicates would be every single category in the pack.  I.e there is no instance where `a.b.c` is duplicated where `a.b` and `a` are as well.

Reply:
as discussed offline: lets add some extra instructions to the user as to what to do when there are duplicates.